packages/pigeon/lib/cpp_generator.dart

```diff
@@ -23,6 +23,24 @@ const String _standardCodecSerializer = 'flutter::StandardCodecSerializer';
 /// The name of the codec serializer.
 const String _codecSerializerName = 'PigeonCodecSerializer';
 
+const String _overflowClassName = '${varNamePrefix}CodecOverflow';
```

Comment:
I changed the prefix, let me know if that is enough or if I need to `internal`ize it still.

Reply:
It would be more idiomatic C++ to put things that have to be in the header but aren't intended for direct use in an `internal` namespace, but it's not a big deal, so we can leave this as-is.